pipelines/feature_generation/README.md

```diff
@@ -0,0 +1,18 @@
+# Generate feature index
+## make sure you have the requisite data
+you should download a catalogue snapshot
+```
+wget https://data.wellcomecollection.org/catalogue/v2/works.json.gz
```

Comment:
Worth pointing out that this is waaaayyyy out of date, the snapshot generator doesn't work